pymorphy3/analyzer.py

```diff
@@ -252,7 +248,7 @@ def _lang_default_config(self):
         aliases = {'ru-old': 'ru'}
         lang = aliases.get(self.lang, self.lang)
         if not hasattr(pymorphy3.lang, lang):
-            warnings.warn("unknown language code: %r" % lang)
+            warnings.warn(f"unknown language code: {repr(lang)}")
```

Comment:
The use of `!s`, `!r`, `!a` is mostly relevant to compatibility issues of Python 2 codes. Its use in Python 3 f-strings are rare. [PEP 498](https://peps.python.org/pep-0498/#s-r-and-a-are-redundant) said that they are redundant, since they can be simply replace by `str()`, `repr()`, and `ascii()`, which is more readable I suppose.

Reply:
@BLKSerene, ok, agreed